packages/desktop-client/src/components/modals/GoalTemplateModal.tsx

```diff
@@ -0,0 +1,88 @@
+import { theme } from '../../style';
+import { Link } from '../common/Link';
+import { Modal, ModalCloseButton, ModalHeader } from '../common/Modal';
+import { Text } from '../common/Text';
+import { View } from '../common/View';
+
+export function GoalTemplateModal() {
+  return (
+    <Modal name="goal-templates">
+      {({ state: { close } }) => (
+        <>
+          <ModalHeader
+            title="Goal Templates"
+            rightContent={<ModalCloseButton onPress={close} />}
+          />
+          <View
+            style={{
+              flexDirection: 'row',
+              fontSize: 13,
+            }}
+          >
+            <Text style={{ textAlign: 'left' }}>
+              <h3>Weekly:</h3>
```

Comment:
Instead of using the h3's directly can we use the <Heading> component provided by react-aria-components https://react-spectrum.adobe.com/react-spectrum/Heading.html#heading